sourcemap/mapper.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 const sourcemapContentSnippetSize = 5
+const sourcemapLineLimit = 256
```

Comment:
I think this would be classified as a breaking change. To avoid that, we would need to have it configurable and unlimited by default.

Reply:
https://github.com/elastic/apm-server/issues/4149
It's too hard for me, so I've created a feature request issue.

Reply:
OK, thanks all the same. If you have any trouble setting up an Ingest pipeline as an alternative, please open a topic at https://discuss.elastic.co/c/apm and someone will try to help out.

Reply:
Thank you, I've deployed some coordinating nodes to avoid crash the data nodes.
I think the OOM happend in the gather phase.